src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -78,7 +83,8 @@ class WorkspaceNewRoomPage extends React.Component {
      */
     submit(values) {
         const policyID = this.props.policies[`${ONYXKEYS.COLLECTION.POLICY}${values.policyID}`];
-        Report.addPolicyReport(policyID, values.roomName, values.visibility);
+        const policyMembers = _.map(_.keys(this.props.allPolicyMembers[`${ONYXKEYS.COLLECTION.POLICY_MEMBERS}${values.policyID}`]), Number);
```

Comment:
I think this would be a bit clearer. 

```suggestion
        const policyMembers = _.map(_.keys(this.props.allPolicyMembers[`${ONYXKEYS.COLLECTION.POLICY_MEMBERS}${values.policyID}`]), accountID => accountID);
```